Extension/Action.js

```diff
@@ -35,249 +35,192 @@ var pingerPonger = {
     }
 };
 
-function validAppId(appId) {
```

Comment:
The diff in this file is substantial, but it looks like things are just being moved around without any changes.